docs/source/posts/2022/2023-02-01-final-report-shivam.rst

```diff
@@ -0,0 +1,406 @@
+
+.. role:: raw-html(raw)
+   :format: html
+
+.. raw:: html
+
+   <center><a href="https://summerofcode.withgoogle.com/programs/2022/projects/ZZQ6IrHq"><img src="https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-logo-horizontal.svg" alt="gsoc" height="50"/></a></center>
+
+.. raw:: html
+
+   <center>
+   <a href="https://summerofcode.withgoogle.com/projects/#6653942668197888"><img src="https://www.python.org/static/community_logos/python-logo.png" height="45"/></a>
+   <a href="https://fury.gl/latest/community.html"><img src="https://python-gsoc.org/logos/FURY.png" alt="fury" height="45"/></a>
+   </center>
+
+
+Google Summer of Code Final Work Product
+========================================
+
+.. post:: February 01 2023
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+-  **Name:** Shivam Anand
+-  **Organisation:** Python Software Foundation
+-  **Sub-Organisation:** FURY
+-  **Project:** `FURY - glTF
+   Intergration <https://github.com/fury-gl/fury/wiki/Google-Summer-of-Code-2022>`__
+
+
+Proposed Objectives
+-------------------
+
+-  Ability to load glTF models
+
+   -  Should be able to load static glTF models
+   -  Should be able to add model to the scene.
+
+-  Exporting scene data as glTF file
+-  Materials & Textures
+
+   -  Textures
+   -  PBRs
+
+-  Animations
+
+   -  Simple Actor animations
+   -  Skinning
+   -  Morphing
+
+-  Stretch Goals
+
+   -  Ability to load ``.glb`` files
+
+Objectives Completed
+--------------------
+
+Loading Static glTF models
+**************************
+
+
+   A glTF file is a JSON like file format containing required data for
+   3D scenes. VTK has two built-in glTF loaders. However, they lack
+   ability to animate and apply materials. Added methods to load binary
+   data and create actors from them. These actors can be added directly
+   to the scene. The glTF class reads texture data from either
+   ``base64`` encoded string or from the image file, and maps the
+   texture to the actor using the given UV data. It is capable of doing
+   the following:
+
+   -  Loading both ``gltf`` and ``glb`` files. Get actors from the
+      model.
+   -  Applying textures and colors from materials.
+   -  Setting cameras if the model contains multiple cameras.
+   -  Apply normals (for a smoother surface).
+
+   .. figure:: https://user-images.githubusercontent.com/74976752/174492510-b9f10816-3058-4a7b-a260-0627406354ba.png
+      :alt: image
+
+
+   *Pull Requests:*
+
+   -  **Importing glTF files: (Merged)**
+      https://github.com/fury-gl/fury/pull/600
+   -  **Loading glTF Demo (with textures) (Merged):**
+      https://github.com/fury-gl/fury/pull/600
```

Comment:
You should avoid tabulation in the whole document. See below how it looks like due to the tabulations: 

![image](https://user-images.githubusercontent.com/23106443/216667663-997d249d-2494-4566-acae-b38446d1b1e5.png)

Reply:
Done @skoudoro , PTAL
Thanks